actions/utils/prerun/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Prerun checks"
+description: "Initialization and checks before running the deploy."
+outputs:
+  result:
+    description: "May be 'skip', to indicate the rest of the workflow should be skipped"
+    value: ${{ steps.cleanup-closed-pr.outputs.result }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
```

Comment:
Is this checking out the code? 

I think the challenge we've seen with closing PRs is that the checkout step fails because the ref branch doesn't exist (the merge and delete happens as the merge action is triggered).

We would need to make the checkout conditional on the event, or have this step come after the cleanup step